source/common/http/conn_manager_impl.h

```diff
@@ -67,7 +67,8 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
                         Random::RandomGenerator& random_generator, Http::Context& http_context,
                         Runtime::Loader& runtime, const LocalInfo::LocalInfo& local_info,
                         Upstream::ClusterManager& cluster_manager,
-                        Server::OverloadManager& overload_manager, TimeSource& time_system);
+                        Server::OverloadManager& overload_manager, TimeSource& time_system,
+                        bool use_proactive_draining = true);
```

Comment:
Can you comment on use of an argument instead of reading the envoy.reloadable_features.http_conn_manager_proactive_drain runtime feature inside the constructor?  Reading the runtime feature in the constructor seems more consistent.

Reply:
By admin handlers I'm referring to the set of HTTP endpoints on Envoy's admin interface (`/config_dump`, `/stats`, etc).